src/components/encoding-pane/property-editor-schema.ts

```diff
@@ -81,34 +109,65 @@ const POSITION_FIELD_TEMPORAL_INDEX = {
   'Axis': ['orient', 'title'].map(p => ({prop: 'axis', nestedProp: p}))
 };
 
-// Capitalize first letter for aesthetic purposes in form
-function toTitleCase(str: string) {
-  return str.replace(/\w\S*/g, txt => {
-    return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
-  });
-}
+const COLOR_CHANNEL_FIELD_INDEX = {
```

Comment:
Why don't these channel have "common" tab like position?

Reply:
Wasn't sure which ones would be "most common". @felixcodes any ideas?

Reply:
Legend title, scale range or scheme